lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -0,0 +1,302 @@
+;(function (define, undefined) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone', 'js/vendor/backbone-super',
+    ], function (gettext, $, _, Backbone) {
+
+        var messageRevertDelay = 4000;
+        var AccountSettingsFieldViews = {};
+
+        AccountSettingsFieldViews.FieldView = Backbone.View.extend({
```

Comment:
I love this field view code! I'm wondering whether it could be abstracted slightly so that it isn't specific to account settings, but could be used to edit any AJAX-based resource. I could see us using this in a lot of other situations. It also is very close to the Studio XBlock editing infrastructure, but merging those two is a much bigger challenge. @cahrens what do you think?